docs: Use sphinx_rtd_theme

Upstream has changed its docs theme, but we don't have this additional
theme packaged in MacPorts and neither do we want the additional build
depdendency. Patch the docs config file to switch back to the previous
theme instead.

Upstream-Status: Inappropriate [configuration]

--- docs/conf.py.orig	2025-07-15 10:27:32
+++ docs/conf.py	2025-07-15 10:28:51
@@ -99,10 +99,10 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-import guzzle_sphinx_theme
+import sphinx_rtd_theme
 
-html_theme_path = guzzle_sphinx_theme.html_theme_path()
-html_theme = 'guzzle_sphinx_theme'
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'sphinx_rtd_theme'
 
 
 def set_rst_settings(app):
@@ -260,7 +260,7 @@
     'sphinx.ext.coverage',
     'sphinx.ext.viewcode',
     'sphinxcontrib.jquery',  # jquery is not included anymore by default
-    'guzzle_sphinx_theme',  # register the theme as an extension to generate a sitemap.xml
+    'sphinx_rtd_theme',  # register the theme as an extension to generate a sitemap.xml
 ]
 
 extlinks = {
